### PR TITLE
Update php-cs-fixer to 2.17.2

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update php-cs-fixer to 2.17.2. This brings some improved rules, so the PR fixes the code when it's relevant. I also disable one new option of PHP-CS-Fixer. I'm not against enabling it but it means a very big PR. One thing at a time 😄 .
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22549)

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -43,6 +43,7 @@ return PhpCsFixer\Config::create()
         'self_accessor' => false,
         'yoda_style' => null,
         'single_line_throw' => false,
+        'no_alias_language_construct_call' => false,
```
Comment (matks) [inline .php_cs.dist:46]: This is a new [rule](no_alias_language_construct_call) `no_alias_language_construct_call` that replace all `die()` by `exit()`.

This can be discussed and maybe enabled in another PR. But enabling it means changing thousand of files so I'm not sure this is valuable.

Comment (matks) [reply .php_cs.dist:46]: So I disable it for now.

File: composer.lock [hunk 2]
```diff
@@ -10427,6 +10430,7 @@
         "ext-intl": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
+        "ext-simplexml": "*",
```
Comment (matks) [inline composer.lock:10433]: Little extra 👍

File: src/Core/Form/IdentifiableObject/DataHandler/AddressFormDataHandler.php [hunk 3]
```diff
@@ -61,7 +61,7 @@ public function __construct(CommandBusInterface $commandBus, CustomerDataProvide
     }
 
     /**
-     * {@inheritdoc]
+     * {@inheritdoc}]
```
Comment (Progi1984) [inline src/Core/Form/IdentifiableObject/DataHandler/AddressFormDataHandler.php]:
```suggestion
     * {@inheritdoc}
```

File: src/Core/Form/IdentifiableObject/DataHandler/ManufacturerAddressFormDataHandler.php [hunk 4]
```diff
@@ -51,7 +51,7 @@ public function __construct(CommandBusInterface $commandBus)
     }
 
     /**
-     * {@inheritdoc]
+     * {@inheritdoc}]
```
Comment (Progi1984) [inline src/Core/Form/IdentifiableObject/DataHandler/ManufacturerAddressFormDataHandler.php]:
```suggestion
     * {@inheritdoc}
```

File: .php_cs.dist (hunk 1 above)
Comment (Progi1984) [reply .php_cs.dist:46]: Ok for me.

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks
